templates/default/ndbd-start.sh.erb

```diff
@@ -8,11 +8,17 @@ if [ "X$USERID" != "X<%= node['ndb']['user'] %>" ]; then
    exit -3
 fi  
 
-if [ "${NDBD_INITIAL_RESTART}" = "true" ]; then
+INIT_ARG=
+if [ "${NDBD_INITIAL_RESTART}" = "true" || -n "${INITIAL_START}" ]; then
```

Comment:
The correct syntax is either `[[ "${NDBD_INITIAL_RESTART}" = "true" || -n "${INITIAL_START}" ]]` note the double square brackets or `[ "${NDBD_INITIAL_RESTART}" = "true" ] || [ -n "${INITIAL_START}" ]`